internal/common/helpers.go

```diff
@@ -36,3 +38,51 @@ func IsStringInSortedSlice(slice []string, s string) bool {
 	}
 	return false
 }
+
+// DataSizeToUint64 converts a size specified as a string in KB/KiB/MB/etc. to
+// a number of bytes represented by uint64.
+func DataSizeToUint64(size string) *uint64 {
```

Comment:
Returning `(uint64, error)` feels more natural to me but that might be just my opinion. :D